src/client/src/services/model/connectionType.js

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
@raybooysen do you have comments on this 'enum' styling as opposed to an object with static props?

Am I right in thinking this encourages the use of magic strings and should be avoided?